tests/test_html5lib_shim.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from bleach import html5lib_shim
+
+
+@pytest.mark.parametrize('data, expected', [
+    # Strings without character entities pass through as is
+    ('', ''),
+    ('abc', 'abc'),
+
+    # Handles character entities--both named and numeric
+    ('&nbsp;', u'\xa0'),
+    ('&#32;', ' '),
+    ('&#x20;', ' '),
+
+    # Handles ambiguous ampersand
+    ('&xx;', '&xx;'),
+
+    # Handles multiple entities in the same string
+    ('this &amp; that &amp; that', 'this & that & that'),
+])
+def test_convert_entities(data, expected):
+    assert html5lib_shim.convert_entities(data) == expected
```

Comment:
Possible fixme: more of the functions should have tests in this module. They're implicitly tested via other tests in `test_clean`, but maybe it's prudent to test them explicitly here?

Reply:
If `convert_entities` isn't a public API, I'd say keep it DRY and don't duplicate the tests as long as coverage stays the same. Otherwise, might be worth duplicating the tests.

Reply:
That makes sense. I'll think about it a bit more and see if I can think of a compelling reason one way or the other.